.github/workflows/sprint-starter.yml

```
@@ -0,0 +1,264 @@
+        options:
+          - 'start'
+          - 'end'
+          - 'report'
+
+jobs:
+  manage-sprint:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: read
+      projects: write
```

actionlint (1.7.7)

unknown permission scope "projects". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)